documentation/Functions.md

```diff
@@ -244,11 +249,24 @@ The option to insert at a linenumber uses a placeholder inserted by a script and
 ***
 
 ### createFolder <a name="createFolder"></a>
+This function is used to create a new folder.
 #### parameter 
-1. Path of the folder to be created, relative to the workspace directory. Subdirectories are also created.
+This function has only one parameter.
```

Comment:
For some functions you are using  `This function consist of two parameters.` this should be consistent.